lib/nerves_bootstrap.ex

```diff
@@ -30,4 +37,30 @@ defmodule Nerves.Bootstrap do
       _e -> :noop 
     end
   end
+
+  @doc """
+  Merge existing aliases with the default nerves_bootstrap aliases
+  """
+  def merge_aliases(aliases) do
+    aliases
+    |> append("deps.get", "nerves.deps.get")
+    |> prepend("deps.precompile", "nerves.precompile")
+    |> append("deps.loadpaths", "nerves.loadpaths")
+  end
+  
+  defp append(aliases, a, na) do
+    unless Enum.member?(aliases, a) do
+      Keyword.update(aliases, String.to_atom(a), [a, na], &(&1 ++ [na]))
+    else
+      aliases
+    end
+  end
+  
+  defp prepend(aliases, a, na) do
+    unless Enum.member?(aliases, a) do
```

Comment:
Will `Enum.member?(aliases, a)` ever return true? Is that even valid?

Reply:
you're right, it needs to be an atom